invenio_accounts/profiles/schemas.py

```diff
@@ -33,3 +41,4 @@ class UserPreferencesSchema(Schema):
 
     visibility = fields.String(validate=validate_visibility)
     email_visibility = fields.String(validate=validate_visibility)
+    preferences_locale = fields.String(validate=validate_preferences_locale)
```

Comment:
Shall we remove the `preferences` prefix everywhere?
```suggestion
    locale = fields.String(validate=validate_locale)
```